tests/autohint_test.py

```diff
@@ -81,7 +80,7 @@ def test_basic_hinting(font_filename, opt):
         actual_path = get_temp_file_path()
 
     diff_mode = []
-    if tail == '.cff':
+    if tail in {'.cff', '.pfb'}:
```

Comment:
is there a particular reason to use a dict instead of a tuple?

Reply:
That's a set, not dict:
```
>>> s = {1, 2}
>>> type(s)
<class 'set'>
```